packages/xo-server-sdn-controller/src/index.js

```diff
@@ -638,11 +854,28 @@ class SDNController extends EventEmitter {
       })
     )
 
+    const crossPoolNetwork = find(this._crossPoolNetworks, crossPoolNetwork =>
+      crossPoolNetwork.networks.includes(network.$ref)
+    )
     if (newCenter === undefined) {
       log.error('No available host to elect new star-center', {
         network: network.name_label,
         pool: network.$pool.name_label,
       })
+
+      // Re-elect a cross-pool center if needed
+      if (
+        crossPoolNetwork !== undefined &&
+        crossPoolNetwork.poolCenter === network.$pool.$ref
```

Comment:
You can combine this test: `crossPoolNetwork?.poolCenter === network.$pool.$ref`.